letsencrypt-auto-source/letsencrypt-auto.template

```diff
@@ -319,6 +319,32 @@ else
   BOOTSTRAP_VERSION="Unknown 1"
 fi
 
+PrevBootstrapVersion() {
+  if [ -f $VENV_BOOTSTRAP_VERSION ]; then
+    cat $VENV_BOOTSTRAP_VERSION
+  elif grep -Fqx "$BOOTSTRAP_VERSION" << "UNLIKELY_EOF"
+NO_BOOTSTRAP 1
+BootstrapDebCommon 1
```

Comment:
I was actually going to suggest giving all the existing bootstrap scripts version 0, since it's more "special" to human eyes and people will probably give version 1 to new scripts no matter what our documentation says. The fact that we arrived at the same solution makes me lean further in that direction. That, combined with --no-bootstrap genuinely sidestepping the bootstrapping logic, is my favored approach.